src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    current = propertyValues[i];
+
+    for (let j = i + 1; j < propertyValues.length; j++) {
+      if (current === propertyValues[j]) {
+        return null;
+      }
+    }
+  }
+
+  for (const key in robot) {
+    inverse[robot[key]] = key;
+  }
```

Comment:
In this loop, you are inverting the object. You can also check for duplicates here. If the key (which is the value in the original object) already exists in the inverted object, it means that there is a duplicate value in the original object. In this case, you can return null immediately. This will make your function more efficient, as it will only need to loop through the object once.